design/FeaturesWithLimits.md

```diff
@@ -0,0 +1,124 @@
+# Features that add new limits
+
+Some new features add associated limits. This document records a design policy for such features.
+
+## Common behaviors
+
+- `{ newLimit: undefined }` is allowed even if `newLimit` is unknown. (It has no effect.)
+- Limits entries associated with features have these Adapter Capability Guarantees:
+  - They are `>= default` if the feature is available on the adapter.
```

Comment:
This should be classed by type of limit. Alignments for example should be `<= default`.

Reply:
Agreed. We can probably lean on the concept of "better" as defined in the spec here.
```suggestion
  - They are [better than](https://gpuweb.github.io/gpuweb/#limit-better) or equal to `default` if the feature is available on the adapter.
```